core-bundle/contao/drivers/DC_Table.php

```diff
@@ -2360,7 +2360,7 @@ public function editAll($intId=null, $ajaxId=null)
 		// Add fields
 		$fields = $session['CURRENT'][$this->strTable] ?? array();
 
-		$blnAddJumpTarget = \count($ids) < 25;
+		$blnAddJumpTarget = \count($ids) < min(Config::get('resultsPerPage') ?? 50, 50);
```

Comment:
```suggestion
		$blnAddJumpTarget = \count($ids) < min(Config::get('resultsPerPage') ?? 30, 50);
```

The default is `30`, isn't it?

Reply:
Fixed in af5bc49fecc616643c892d1cd950ece0d6fb237b